backend/routes/api/memberships.js

```diff
@@ -141,4 +135,115 @@ router.post("/:groupId", requireAuth, async (req, res, next) => {
   }
 });
 
+// ---------------- UPDATE ENDPOINTS -------------------------
+
+// updated member's status
+router.put('/:groupId', requireAuth, async (req, res, next) => {
+    const { groupId } = req.params
+    const { memberId, status } = req.body
+
+    // find group by groupId then check if it exist
+    const group = await Group.findOne({where: { id: groupId }})
+    if (!group) {
+        const err = new Error("Group couldn't be found")
+        err.status = 404
+        return next(err)
+    }
+
+    // if status is pending, throw error
+    if (status === 'pending') {
+        const err = new Error('Validations Error')
+        err.status = 400
+        err.errors = { status: "Cannot change a membership status to pending"}
+        return next(err)
+    }
+
+    const coHost = await Membership.findOne({where: { memberId: req.user.id, groupId: groupId }})
+    const member = await Membership.findOne({where: { memberId: memberId, groupId } })
+
+    // check if user is a member
+    if (!member) {
+        const err = new Error('Validation Error')
+        err.status = 400
+        err.errors = { status: "User couldn't be found"}
```

Comment:
For this error, I'd check if a User exists given the memberId ( query for a User )